datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/OwnerUtils.java

```diff
@@ -300,8 +301,8 @@ public static void addCreatorAsOwner(
     }
   }
 
-  public static String mapOwnershipTypeToEntity(OwnershipType type) {
-    final String typeName = SYSTEM_ID + type.name().toLowerCase();
```

Comment:
why make this change here?

Reply:
Because `OwnershipType` is a graphQL generated class. Wanted to use the same method for the PDL-generated equivalent class, as that is what was being used in the `OwnerUpdateMapper.java` file.